src/llm/llm_calculator.proto

```diff
@@ -41,4 +41,8 @@ message LLMCalculatorOptions {
     optional string device = 8 [default = "CPU"];
 
     optional string plugin_config = 9 [default = ""];
+
+    optional uint32 best_of_limit = 10 [default = 20];
+
+    optional uint32 max_tokens_limit = 11 [default = 4096];

```

Comment:
Does mediapipe allow nested structure in options? Maybe we could get it under some `limits` map?